Index: src/com/company/Reservation.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/company/Reservation.java	(date 1636585372906)
+++ src/com/company/Reservation.java	(date 1636585372906)
@@ -0,0 +1,27 @@
+package com.company;
+
+import java.util.ArrayList;
+
+public class Reservation {
+    private int ID, custID, signID, statusID, amount;
+    private String requestedOn, lastActivity;
+
+    /**
+     * @param ID reservation ID number for database access
+     * @param custID customer ID number
+     * @param signID sign ID number
+     * @param statusID status ID number
+     * @param amount quantity of signs being reserved
+     * @param requestedOn the date when the reservation was submitted
+     * @param lastActivity date of the client's last reservation
+     */
+    public Reservation(int ID, int custID, int signID, int statusID, int amount, String requestedOn, String lastActivity){
+        this.ID = ID;
+        //this.custID = custID;
+        //this.signID = signID;
+        //this.statusID = statusID;
+        this.amount = amount;
+        this.requestedOn = requestedOn;
+        this.lastActivity = lastActivity;
+    }
+}
